<launch>
    <param name="robot_frame" value="base_link"/>
    <param name="world_frame" value="map"/>
    <param name="odom_frame"  value="odom"/>
    <param name="num_landmarks" type="int" value="15"/>

    <!-- ekf input topic -->
    <node pkg="kamayuc_localization" type="map_artag_ekf" name="map_artag_ekf">
    </node>
</launch>
